client/app/views/components/_statusView.scss

```diff
@@ -17,6 +17,7 @@
   &[data-status=success] { @include progress-bar(#2e7d32); }
   &[data-status=failure] { @include progress-bar(#c62828); }
   &[data-status=running] { @include progress-bar(#546e7a); }
+  &[data-status=unknown] { @include progress-bar(#87a1ad); }
```

Comment:
Color literals like `#87a1ad` should only be used in variable declarations; they should be referred to via variable everywhere else.